devnet/terra-devnet.yaml

```diff
@@ -119,7 +119,7 @@ spec:
             - name: POSTGRES_DB
               value: fcd
       restartPolicy: Always
-  serviceName: terra-fcd
```

Comment:
how did this work before?

Reply:
I don't think it did. It might have connected to the terra classic service. Stumbled across this while trying to figure out why one of the terra2 containers was crash-looping (this was not it).